modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

```diff
@@ -134,20 +151,34 @@ metadata:
   namespace: openshift-user-workload-monitoring
 data:
   config.yaml: |
-    *prometheus*:
+    prometheus:
       volumeClaimTemplate:
         spec:
-          storageClassName: *local-storage*
+ifndef::openshift-dedicated,openshift-rosa[]
+          storageClassName: local-storage
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+          storageClassName: gp3
+endif::openshift-dedicated,openshift-rosa[]
           resources:
             requests:
-              storage: *40Gi*
+              storage: 40Gi
 ----
 +
+ifndef::openshift-dedicated,openshift-rosa[]
 In the above example, the storage class created by the Local Storage Operator is called `local-storage`.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+The above example uses the `gp3` storage class, which is the default storage class for AWS Elastic Block Store (EBS) volumes.
```

Comment:
```suggestion
The above example uses the `gp3` storage class.
```
I suggest we don't need to mention the default storage class, in case of the default storage class changed in the future

Reply:
Done.